test/metabase/pulse/dashboard_subscription_test.clj

```diff
@@ -580,30 +555,22 @@
                 (mt/summarize-multipart-single-email email #"(?s)Test card.*State.*CA, NY, and NJ")))))
 
      :slack
-     (fn [{:keys [card-id dashboard-id]} [pulse-results]]
+     (fn [{:keys [card-id dashboard-id]} [message]]
        (testing "Dashcard with inline parameters shows State parameter in dashboard fields"
-         (is (= {:channel-id "#general"
-                 :attachments
-                 [{:blocks [{:type "header", :text {:type "plain_text", :text "Aviary KPIs", :emoji true}}
-                            {:type "section",
-                             :fields [{:type "mrkdwn", :text "*Quarter and Year*\nQ1, 2021"}]}
-                            {:type "section",
-                             :fields (append-subscription-branding-content [{:type "mrkdwn",
-                                                                             :text (str "<https://testmb.com/dashboard/"
-                                                                                        dashboard-id
-                                                                                        "?state=CA&state=NY&state=NJ&quarter_and_year=Q1-2021|*Sent from Metabase Test by Rasta Toucan*>")}])}]}
-                  {:title             pulse.test-util/card-name
-                   :rendered-info     {:attachments false, :content true, :render/text true},
-                   :inline-parameters [{:value ["CA" "NY" "NJ"]
```

Comment:
What's happened to the inline parameters?

Reply:
oh yea, good catch. it should be fixed in https://github.com/metabase/metabase/pull/60838/commits/0305cd81a6ad36bd56847fe33c27620f0d1eabc3